cmd/ocm-backplane/login/login.go

```diff
@@ -119,6 +126,18 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 			"Name": clusterName}).Infoln("Management cluster")
 	}
 
+	if args.service {
```

Comment:
It's good to add a unit test to cover the `service`  cluster scenario.
eg : management cluster test scenario : https://github.com/openshift/backplane-cli/blob/3da00fc2a1e6b3392083f7e3d2f0202b6a9cc3ce/cmd/ocm-backplane/login/login_test.go#L214

Reply:
Thanks, the related testing is added